internal/debugger/tarball.go

```diff
@@ -105,100 +121,27 @@ func CreateTarball(ssm bool) {
 			fmt.Println("Error adding answers to tarball:", err)
 		}
 	}
 
 	fmt.Println("Tarball created successfully at:", outputPath)
 }
 
-func Triage() Answers {
-	answers := Answers{}
-	reader := bufio.NewReader(os.Stdin)
-
-	fmt.Println("Please answer these questions to better assist with your issue:")
-
-	fmt.Println("Is this issue once off, intermittent, or consistently happening right now? (o/i/c): ")
-	occurence, err := reader.ReadString('\n')
-	if err == nil {
-		answers.Occurence = strings.TrimSpace(occurence)
-	}
-
-	fmt.Print("Has anything changed in the environment recently? (y/n): ")
-	envChange, err := reader.ReadString('\n')
-
-	if err == nil {
-		answers.EnvironmentChange = strings.TrimSpace(envChange)
-
-		if strings.ToLower(answers.EnvironmentChange) == "y" {
-			fmt.Print("Please describe what changed and when: ")
-			envChangeDesc, err := reader.ReadString('\n')
-			if err == nil {
-				answers.EnvironmentChangeDesc = strings.TrimSpace(envChangeDesc)
-			}
-		}
-	}
+func Triage() string {
 
-	fmt.Print("Is there any additional information you would like to add? ")
-	addInfo, err := reader.ReadString('\n')
-
-	if err == nil {
-		answers.AddInfo = strings.TrimSpace(addInfo)
-	}
-
-	return answers
-}
-
-func writeTriage() string {
-
-	answers := Triage()
-
-	// Create answers content
-	answersContent := "CloudWatch Agent Debugging Information\n"
-	answersContent += "===================================\n\n"
-	answersContent += "Q: Is this issue once off, intermittent, or consistently happening right now?\n"
-
-	// Formatting answers
-	occurrenceAnswer := ""
-	switch strings.ToLower(answers.Occurence) {
-	case "o":
-		occurrenceAnswer = "Once off"
-	case "i":
-		occurrenceAnswer = "Intermittent"
-	case "c":
-		occurrenceAnswer = "Consistently happening"
-	default:
-		occurrenceAnswer = answers.Occurence
-	}
-	answersContent += "A: " + occurrenceAnswer + "\n\n"
-
-	answersContent += "Q: Has anything changed in the environment recently?\n"
-	envChangeAnswer := ""
-	if strings.ToLower(answers.EnvironmentChange) == "y" {
-		envChangeAnswer = "Yes"
-		answersContent += "A: " + envChangeAnswer + "\n\n"
-		answersContent += "Q: Please describe what has changed and when:\n"
-		answersContent += "A: " + answers.EnvironmentChangeDesc + "\n\n"
-	} else {
-		envChangeAnswer = "No"
-		answersContent += "A: " + envChangeAnswer + "\n\n"
-		answersContent += "Q: Please describe what has changed and when:\n"
-		answersContent += "A: N/A\n\n"
-	}
-
-	answersContent += "Q: Is there any additional information you would like to add?\n"
-	answersContent += "A: " + answers.AddInfo + "\n\n"
-
-	return answersContent
+	answers := utils.RunTriage()
+	formattedAnswers := utils.FormatReport(answers)
+	return formattedAnswers
 
 }
 
-func addStringToTarball(tarWriter *tar.Writer, content, tarPath string) error {
+func addTriageToTarball(tarWriter *tar.Writer, content, tarPath string) error {
```

Comment:
you are saying addTriage but its not part of the parameter, I don't understand what this function does